LoggingKata.Test/TacoParserTests.cs

```diff
@@ -5,25 +5,38 @@ namespace LoggingKata.Test
 {
     public class TacoParserTests
     {
-        [Fact]
-        public void ShouldDoSomething()
-        {
-            // TODO: Complete Something, if anything
-        }
-
         [Theory]
-        [InlineData("Example")]
+        [InlineData("1,1,name")]
+        [InlineData("12.89,98.321, not real")]
+        [InlineData("-17.58913, -100.00487, middle of no where")]
         public void ShouldParse(string str)
         {
-            // TODO: Complete Should Parse
+            //arange 
+            TacoParser tacoParser = new TacoParser();
+
+            //act
+            var tacoBell = tacoParser.Parse(str);
+
+            //assert
+            Assert.NotNull(tacoBell)

```

Comment:
Are there perhaps some other assertions that could be made to verify a successful parse?